### DISCUSSION TITLE
@formily/antd cascader previewText 没有执行 displayRender 方法，那怎么自定义渲染呢？

### DISCUSSION THREAD

**Discussion by qiphon** [alibaba/formily#3740] [Q&A 2.x]
![image](https://user-images.githubusercontent.com/24565078/221180514-2633d951-24ab-4057-9e7c-bf2a7021a445.png)

**Comment by xiaosi-ss** [comment]
可以覆写previewText
import { PreviewText } from '@formily/antd';

大概这种 
const PreviewTextInput: FC<InputProps> = ({ value, ...props }) => {
  return <PreviewText.Input {...props} value={value} />;
};

用的时候mapReadPretty
